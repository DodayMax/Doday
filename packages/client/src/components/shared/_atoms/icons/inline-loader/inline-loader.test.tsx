import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('InlineLoaderIcon', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.InlineLoader />);

    expect(icon).toMatchSnapshot();
  });
});
