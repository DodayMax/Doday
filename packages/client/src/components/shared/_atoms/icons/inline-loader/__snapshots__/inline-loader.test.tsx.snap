// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineLoaderIcon renders correctly without props 1`] = `ShallowWrapper {}`;
